bin/deploy-to-github-pages.js

```diff
@@ -48,6 +48,13 @@ function cleanOptions(options) {
   }, {});
 }
 
+function cleanBranchName(branchName) {
+  if (branchName.includes('/')) {
```

Comment:
This will only replace the first one, right? If so, should we replace all?

Reply:
Nice catch, I'll update